src/schema/comments.ts

```diff
@@ -389,17 +565,25 @@ export const resolvers: IResolvers<any, Context> = {
 
       try {
         const comment = await ctx.con.transaction(async (entityManager) => {
-          const comment = await entityManager.getRepository(Comment).save(
-            entityManager.getRepository(Comment).create({
-              id: shortid.generate(),
-              postId,
-              userId: ctx.userId,
-              content,
-            }),
+          const mentions = await getMentions(
+            entityManager,
+            content,
+            ctx.userId,
           );
+          const createdComment = entityManager.getRepository(Comment).create({
+            id: shortid.generate(),
+            postId,
+            userId: ctx.userId,
+            content,
+          });
+          createdComment.mentions = mentions.map((mention) => mention.username);
+          const comment = await entityManager
+            .getRepository(Comment)
+            .save(createdComment);
           await entityManager
             .getRepository(Post)
             .increment({ id: postId }, 'comments', 1);
+          await saveCommentMentions(entityManager, comment.id, mentions);
```

Comment:
I see that this looks quite similar to `commentOnComment`. Maybe we can reuse some of the stuff?